gpu.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG BASE_TAG=staging
 
 FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04 AS nvidia
-FROM gcr.io/kaggle-images/python-tensorflow-whl:2.1.0-py37-2 as tensorflow_whl
+FROM gcr.io/kaggle-images/python-tensorflow-whl:2.2.0-py37 as tensorflow_whl
```

Comment:
Nit: should we substitute `${TENSORFLOW_VERSION}` for `2.2.0` here to help keep these in sync?

Reply:
Not sure I follow. `${TENSORFLOW_VERSION}` is not defined in gpu.Dockerfile.

The only reason we use it in `Dockerfile` is because we need it in two places:
- To set the `tensorflow-version` docker LABEL.
- For the TF whl image name.

In the GPU image, the `tensorflow-version` docker LABEL is inherited from the base cpu image. This means we only need the TensorFlow version in one place.

Reply:
Nevermind. I didn't notice that this switched over from Dockerfile to gpu.Dockerfile between these two changes.